src/components/Explore/Explore.tsx

```diff
@@ -170,9 +177,9 @@ const Explore: React.FunctionComponent<{
 
   // Resets the state when navigating locations
   useEffect(() => {
-    setSelectedLocations([currentLocation]);
+    setSelectedLocations(currentLocations);
     setCurrentMetric(defaultMetric);
-  }, [currentLocation, defaultMetric]);
+  }, [currentLocations, defaultMetric]);
```

Comment:
what does "navigating locations" mean?

Reply:
When users click other states or counties either in the compare table or the mini map, the page updates without doing a full reload, so we need to reset the selected locations and default metric so they don't see the selections of their previous location in the new location.

I will update the comment to clarify 👍